### ISSUE TITLE
<leader> ff no longer fuzzy finding since update to v3

### ISSUE THREAD

**Issue by Callumk7** [siduck76/NvChad#3094]
Hello,

Since updating to v3, <leader>ff no longer fuzzy finds files, instead only displays exact matches for file names

Is it possible to switch this back?

Thanks

**Comment by siduck** [comment]
![image](https://github.com/user-attachments/assets/eec845b2-85e9-452d-b492-0811586b8420)

that mapping has not been changed since april. Which fuzzy stuff are you talking about :thinking: . Show before/after, then i'll look into this !

**Comment by Callumk7** [comment]
Hello, yep I totally jumped the gun, assumed this was a different picker. 

I updated my plugins earlier and I lost fzf functionality in most of the file pickers in telescope - which I don't think I had to explicitly set in the previous version. 

I can get it back by adding https://github.com/nvim-telescope/telescope-fzf-native.nvim and requiring it in my init.lua. - I am curious where it got unwired in the upgrade though. 

Thanks for the quick reply :)
